lib/components/molecules/QuoteMedia/tests/QuoteMedia.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Blockquote from '../../../atoms/Blockquote/Blockquote';
+import Heading from '../../../atoms/Heading/Heading';
+import QuoteMedia from '../QuoteMedia';
+
+describe('<QuoteMedia />', () => {
+  let QuoteMediaComponent = '';
+  beforeEach(() => {
+    QuoteMediaComponent = shallow(<QuoteMedia />);
```

Comment:
This is wrong Test Environment, You should test the component as if its practically going to be used. Give the sample data.